manager/integration/tests/test_settings.py

```diff
@@ -653,11 +659,82 @@ def test_setting_backing_image_auto_cleanup():  # NOQA
         2. The related backing image file is removed.
         3. The download state of other disks keep unchanged.
            All volumes still work fine.
-    8. Delete all volumes. Verify that all states in the backing image disk map
-       will become terminating first,
-       and all entries will be removed from the map later.
+    8. Delete all volumes. Verify that there will only remain 1 entry in the
+       backing image disk map
     9. Delete the backing image.
     """
+    # Step 1
+    update_setting(client,
+                   "backing-image-cleanup-wait-interval",
+                   BACKING_IMAGE_CLEANUP_WAIT_INTERVAL)
+
+    # Step 2
+    create_backing_image_with_matching_url(
+            client, BACKING_IMAGE_NAME, BACKING_IMAGE_QCOW2_URL)
+
+    # Step 3
+    volume_names = [
+        volume_name + "-1",
+        volume_name + "-2",
+        volume_name + "-3"
+    ]
+
+    for volume_name in volume_names:
+        volume = create_and_check_volume(
+                            client, volume_name, 3,
+                            str(BACKING_IMAGE_EXT4_SIZE),
+                            BACKING_IMAGE_NAME)
+
+    # Step 4
+    lht_host_id = get_self_host_id()
+    for volume_name in volume_names:
+        volume = client.by_id_volume(volume_name)
+        volume.attach(hostId=lht_host_id)
+        wait_for_volume_healthy(client, volume_name)
+        assert volume.backingImage == BACKING_IMAGE_NAME
+
+    backing_image = client.by_id_backing_image(BACKING_IMAGE_NAME)
+    assert len(backing_image.diskFileStatusMap) == 3
+    for disk_id, status in iter(backing_image.diskFileStatusMap.items()):
+        assert status.state == "ready"
+        assert disk_id != ''
+
+    backing_images_in_disk = os.listdir("/var/lib/longhorn/backing-images")
+    assert len(backing_images_in_disk) == 1
+
+    # Step 5
+    replica_node_soft_anti_affinity_setting = \
+        client.by_id_setting(SETTING_REPLICA_NODE_SOFT_ANTI_AFFINITY)
+    client.update(replica_node_soft_anti_affinity_setting, value="true")
+
+    current_host = client.by_id_node(id=lht_host_id)
+    client.update(current_host, allowScheduling=False)
+    wait_for_node_update(client, lht_host_id, "allowScheduling", False)
+
+    # Step 6
+    for volume_name in volume_names:
+        volume = client.by_id_volume(volume_name)
+        for replica in volume.replicas:
+            if replica.hostId == lht_host_id:
+                replica_name = replica.name
+        volume.replicaRemove(name=replica_name)
+
+    time.sleep(int(BACKING_IMAGE_CLEANUP_WAIT_INTERVAL))
```

Comment:
Wait... This is exactly the current backing image cleanup wait interval. After the step, the disk file on the test node may be removed. In brief, you shouldn't wait for so long here.

Reply:
I reduced `BACKING_IMAGE_CLEANUP_WAIT_INTERVAL` to 5, thank you

Reply:
No. I mean, in this line, the wait interval should be smaller than the value of setting `backing-image-cleanup-wait-interval`. So that the auto cleanup won't be triggered and the backing image disk map still holds 3 files.
On the other hand, **all** the **below** wait intervals should be greater than the updated value of the setting, which is "1".

Reply:
Hi @shuo-wu ,
I changed the sleep time to 1 minute to match the test step to avoid uncertain situation happen for review, thanks

Reply:
It's better to set a smaller value here (50s). The value should be slightly smaller than the setting value. Sorry for the confusion. And remember to update the skeleton description as well.